enterprise/frontend/src/metabase-enterprise/advanced_permissions/components/RoleAccessWarning/RoleAccessWarning.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { t } from "ttag";
+import { Database } from "metabase-types/api";
+import * as Urls from "metabase/lib/urls";
+import Link from "metabase/core/components/Link";
+import { RoleAttributeMappingAlert } from "./RoleAccessWarning.styled";
+
+interface RoleAccessWarningProps {
+  database: Database;
+}
+
+const RoleAccessWarning = ({ database }: RoleAccessWarningProps) => {
+  const databaseUser = database.details.user;
+
+  const text =
+    databaseUser == null
```

Comment:
Why the use of `==` instead of `===` here? are we confident that a no user db will always have a `null` user ? or should we just check truthiness?

There's also `isEmpty` (which checks for non zero falsey values)
https://github.com/metabase/metabase/blob/ea25b6932fa988ad84214a1ed344c024cd2c4753/frontend/src/metabase/lib/validate.js#L6-L7

Reply:
I use a wider check because `==` checks for both null and undefined, which does not hurt even if for now there is always a true `null`. If it changes under any circumstances to `undefined` we would still want to show the message without the database user. In this case `isEmpty` even better, thanks